src/main/resources/application.conf

```diff
@@ -553,9 +553,33 @@ scorex {
     # server answer timeout
     timeout = 30s
   }
+}
+
 
+akka {
+   actor {
+     default-dispatcher {
+       type = Dispatcher
+       executor = "thread-pool-executor"
```

Comment:
Are we sure that the default dispatcher should be `thread-pool-executor` which is used for blocking-io? And not `fork-join-executor` ? It would seem reasonable to have both kinds

Reply:
Imho all async barrier crossings that akka does happen on this dispatcher, so it really should has `fork-join-executor`.

Reply:
reverted no default settings